urwid/widget/__init__.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import annotations
+
+from .attr_map import AttrMap, AttrMapError
+from .attr_wrap import AttrWrap
+from .bar_graph import BarGraph, BarGraphError, BarGraphMeta, GraphVScale, scale_bar_values
+from .big_text import BigText
+from .box_adapter import BoxAdapter, BoxAdapterError
+from .columns import Columns, ColumnsError
+from .constants import (
+    RELATIVE_100,
+    Align,
+    Sizing,
+    VAlign,
+    WHSettings,
+    WrapMode,
+    normalize_align,
```

Comment:
normalize_* / simplify_* are useful in downstream code, but was never exported "public"